apps/condo/domains/ticket/constants/errors.js

```diff
@@ -1,5 +1,29 @@
+const { GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+
+const { WRONG_VALUE } = require('@condo/domains/common/constants/errors')
+
+
 const UNKNOWN_GROUP_BY_FILTER = 'UNKNOWN_GROUP_BY_FILTER'
 
+const INCIDENT_ERRORS = {
```

Comment:
usualy we store errors inside the service code. do you want to change this style?

Reply:
fixed it